test/test_messagebuffer.jl

```diff
@@ -0,0 +1,32 @@
+using QuantumSavory
+using QuantumSavory: tag_types
+using ResumableFunctions, ConcurrentSim
+using Test
+
+net = RegisterNet([Register(3), Register(2), Register(3)])
+env = get_time_tracker(net);
+@resumable function receive_tags(env)
+    while true
+        mb = messagebuffer(net, 2)
+        @yield wait(mb)
+        msg = querydelete!(mb, :second_tag, ❓, ❓)
+        print("t=$(now(env)): query returns ")
```

Comment:
Can we comment this out as well? Currently this is flooding the buildkite log and making is super slow to finish the tests

Reply:
oh, yeah, that is some old silly debug message. The code should definitely never use println as it can not be filtered. At most `@debug` should be used.